content/k8s/fluxcd/helmrepository/_index.md

```diff
@@ -0,0 +1,86 @@
+---
+content_type: project
+flavours:
+- none
+from_repo: k8s/manual-app-deployment/project-overview
+prerequisites:
+  hard:
+  - k8s/manual-app-deployment/project-overview
+  soft:
+  - k8s/fluxcd/kustomization
+ready: true
+submission_type: continue_repo
+tags:
+- kubernetes
+- fluxcd
+title: Helmrepository
+---
+
+## Helmrepository
+
+For flux we need a collection of helmrepositories that pull down helm charts to be available to deploy
+
+create the folder infrastructure/sources along with its kustomization.yaml file
+
+```
+# infrastructure/sources/kustomization.yaml
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: flux-system
+resources:
+  - jetstack.yaml
```

Comment:
File name here mismatches the file below. I'd suggest to remove the `-` from the file below and keep it `jetstack.yaml`.